extensions/coroutines-extensions/src/jvmTest/kotlin/com/squareup/sqldelight/runtime/coroutines/MappingJvmTest.kt

```diff
@@ -90,9 +85,7 @@ class MappingJvmTest {
   private suspend fun Flow<*>.assertInitialAndAsyncNotificationUsesContext() = coroutineScope {
     var seen = 0
 
-    // Using undispatched to ensure this reaches first suspension point.
```

Comment:
This was never needed in the first place, but now it actively breaks the test. The reasons are complicated, which is to say I didn't bother looking into them.